### PR TITLE
Add option to use heroku-deploy-wrap

### PR DESCRIPTION
**Pull request by yuya-takeyama**
@quipper/web-devs @quipper/engineering 

Please review and merge!

### CODE REVIEW

**Comment by kachick** [comment]
Just a question, a simple shell code can not do the role?

e.g https://github.com/kachick/heroku_deploy_and_ensure_report/blob/master/heroku_deploy_and_ensure_report.sh

**Comment by yuya-takeyama** [comment]
@kachick 
Almost looks good.
Does the command shows output periodically (without buffering)?

**Comment by yuya-takeyama** [comment]
To do so, how about using `tee` and checking the content of the file?

File: app/views/scripts/production_deploy.erb [hunk 1]
```diff
@@ -25,6 +25,12 @@ if [ "${HEROKU_APPS}" == "" ]; then
   exit 1
 fi
 
+if [ "${USE_HEROKU_DEPLOY_WRAP}" == "1" ]; then
+  GIT_PUSH="heroku-deploy-wrap -- git push"
```
Comment (banyan) [inline app/views/scripts/production_deploy.erb]: Just question, how can we use [heroku-deploy-wrap](https://github.com/yuya-takeyama/heroku-deploy-wrap) in deploy-support-tools? I mean installation of heroku-deploy-wrap.

**Comment by yuya-takeyama** [comment]
I'll retry without using `heroku-deploy-wrap` since it requires extra dependency for apps.

**Pull request closed without merging**